client/extensions/woocommerce/state/sites/product-categories/test/selectors.js

```diff
@@ -194,4 +224,63 @@ describe( 'selectors', () => {
 			expect( getTotalProductCategories( loadedStateWithUi ) ).to.eql( 2 );
 		} );
 	} );
+
+	describe( '#getProductCategoriesLastPage', () => {
+		test( 'should be null (default) when woocommerce state is not available.', () => {
+			expect( getProductCategoriesLastPage( preInitializedState, {}, 123 ) ).to.eql( null );
+		} );
+
+		test( 'should be null (default) when categories are loading.', () => {
+			expect( getProductCategoriesLastPage( loadingState, {}, 123 ) ).to.eql( null );
+		} );
+
+		test( 'should be 2, the set total, if the categories are loaded.', () => {
+			expect( getProductCategoriesLastPage( loadedState, {}, 345 ) ).to.eql( 2 );
+		} );
+
+		test( 'should be null (default) when categories are loaded only for a different site.', () => {
+			expect( getProductCategoriesLastPage( loadedState, {}, 456 ) ).to.eql( null );
+		} );
+	} );
+
+	describe( '#areProductCategoriesLoadingIgnoringPage', () => {
```

Comment:
Should any of the ignoring page tests include a `page` in the query passed in, just to verify it still works as expected?

Reply:
Good call. Done!